server/lib/notifications.js

```diff
@@ -51,7 +58,7 @@ module.exports = (Sequelize, activity) => {
 
 function publishToGitter(activity, notifConfig) {
   const message = activitiesLib.formatMessageForPublicChannel(activity, 'markdown');
-  if (message) {
+  if (message && process.env.NODE_ENV === 'production') {
```

Comment:
Is this needed? Notify is normally already only called in prod:
https://github.com/OpenCollective/opencollective-api/blob/190521380f85ed80019fef4b2b78d3d84b223706/server/models/Activity.js#L19